### PR TITLE
Refactor and cleanup

### PR DESCRIPTION
**Pull request by patrikbraborec**

### CODE REVIEW

**Review by hkad98 — COMMENTED** [review]
Place __pycache__ directory in .gitignore. These files are irrelevant.

File: data-management/src/fetch_and_store_historical_data.py [hunk 1]
```diff
@@ -4,6 +4,10 @@
 import json
 
 
+def get_api_uri(stock):
+    return '/query?function=TIME_SERIES_DAILY&symbol=' + stock + '&datatype=json'
```
Comment (hkad98) [inline data-management/src/fetch_and_store_historical_data.py:8]: You can simplify this using f-string.
f"/query?function=TIME_SERIES_DAILY&symbol={stock}&datatype=json"

File: data-management/src/computations.py [hunk 2]
```diff
@@ -31,7 +33,9 @@ def store_sma(indexed_df, stock_symbol, metric_name):
     database_conn.autocommit = True
     cur = database_conn.cursor()
     for index, row in indexed_df.iterrows():
-        insert = 'INSERT INTO stocks_technical_analysis (symbol, date, sma5) VALUES (\'{0}\',\'{1}\',{2})'.format(stock_symbol, index.date(), row[metric_name + '_sma5'])
+        insert = 'INSERT INTO stocks_technical_analysis (symbol, date, sma30, cma, ema30) VALUES (\'{0}\',\'{1}\',{2}, {3}, {4})'.format(
+            stock_symbol, index.date(), row[metric_name + '_sma30'], row[metric_name + '_cma'],
+            row[metric_name + '_ema30'])
```
Comment (hkad98) [inline data-management/src/computations.py:38]: Consider using f-strings here as well.

Comment (patrikbraborec) [reply data-management/src/computations.py]: Fixed

File: data-management/src/computations.py [hunk 3]
```diff
@@ -16,7 +16,9 @@ def compute_sma(metric_name, frames):
         )
     )
     indexed_df = indexed_df['first_metric'].to_frame()
-    indexed_df[metric_name + '_sma5'] = indexed_df['first_metric'].rolling(5).mean()
+    indexed_df[metric_name + '_sma30'] = indexed_df['first_metric'].rolling(30).mean()
+    indexed_df[metric_name + '_cma'] = indexed_df['first_metric'].expanding().mean()
+    indexed_df[metric_name + '_ema30'] = indexed_df['first_metric'].ewm(span=30).mean()
```
[the remark this answers is not in the document]
Comment (patrikbraborec) [reply data-management/src/computations.py]: Fixed

**Pull request merged**